src/DataLib/fendl32B_retrofit/reaction_data.py

```diff
@@ -192,7 +192,7 @@ def process_mt_data(mt_dict):
             }
     """
 
-    for MT, data in mt_dict.items():
+    for MT, data in list(mt_dict.items()):
```

Comment:
Why does this need to be a `list`?

Reply:
I read that iterating directly through the `items` in a `dict` and using those to delete `keys` from the `dict` can cause runtime errors, so by creating a separate `list`, any potential errors caused by self-references within the `dict` can be avoided.